src/Linkify/__snapshots__/Linkify.spec.js.snap

```diff
@@ -107,3 +139,30 @@ exports[`Linkify escapes HTML entities 1`] = `
   </div>
 </DocumentFragment>
 `;
+
+exports[`Linkify reformats code blocks 1`] = `
+<DocumentFragment>
+  .c0 {
+  white-space: pre-wrap;
+}
+
+.c0 p:first-child {
+  margin-top: 0;
+}
+
+.c0 div + div {
+  margin-top: 8px;
+}
+
+<div
+    class="re-linkify c0"
+  >
+    <div>
+      dear doctor,
+    </div>
+    <span>
```

Comment:
@kylealwyn are we happy removing the additional newline here? It takes 2 newlines for an indented line to be interpreted as a code block. The code renderer approach only results in one.

Reply:
rendering in a div gives us the top buffer that I think we're looking for